packages/kbn-scout/src/config/config.ts

```diff
@@ -13,15 +13,15 @@ import Path from 'path';
 import { cloneDeepWith, get, has, toPath } from 'lodash';
 import { REPO_ROOT } from '@kbn/repo-info';
 import { schema } from './schema';
-import { ScoutServerConfig } from '../types';
-import { formatCurrentDate, getProjectType } from './utils';
+import { ScoutServerConfig, ScoutTestConfig } from '../types';
```

Comment:
`ScoutServerConfig` is compatible with `kbn-test` functionality and used to start the servers, while `ScoutTestConfig` is for Playwright tests and exposes hosts, credentials, licence and other test-required data.